spec/core_functions/selector/is_superselector/simple/pseudo/selector_arg/has.hrx

```diff
@@ -0,0 +1,47 @@
+<===> superset/input.scss
+a {b: is-superselector(":has(c d, e f, g h)", ":has(c d.i, e j f)")}
+
+<===> superset/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> subset/input.scss
+a {b: is-superselector(":has(c d.i, e j f)", ":has(c d, e f, g h)")}
+
+<===> subset/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> bare_sub/input.scss
+a {b: is-superselector(":has(c d.i, e j f)", "c d, e f, g h")}
```

Comment:
Should this be testing the same set of selectors except bare in the sub?

Reply:
Yeah, good point, that's a better way to structure this.